lib/tasks/spotlight_tasks.rake

```diff
@@ -75,6 +75,12 @@ namespace :spotlight do
     Migration::PageLanguage.run
   end
 
+  desc 'Migrate page\'s FriendlyId::Slug to a scoped page type'
+  task migrate_pages_slug_type: :environment do
+    require 'migration/add_page_type_to_friendly_id_scope'
+    Migration::AddPageTypeToFriendlyIdScope.run
```

Comment:
I'm doing it this way, because if we add a migration in the Spotlight codebase it will get run before Friendly IDs migrations are run and will be problematic for new installs.  Not sure if there is a better way to handle that sort of thing.